src/namespace.ts

```diff
@@ -75,7 +75,7 @@ export async function createNamespaceFromDocumentIfNotExists(doc) {
       const client = replSession.getSession(utilities.getFileType(document));
       if (client) {
         const nsList = await client.listNamespaces([]);
-        if (nsList['ns-list'] && nsList['ns-list'].includes(ns)) {
+        if (nsList && nsList['ns-list'] && nsList['ns-list'].includes(ns)) {
```

Comment:
I think the above two lines could be a bit more simplified with optional chaining (I saw another case where this could be used in this PR too).

```typescript
        const nsList = await client.listNamespaces([])?.nsList;
        if (nsList?.includes(ns)) {
```

Reply:
Though this is arguably more clear, I suppose - just a bit less code.

Reply:
It looked like that at first, but then I changed it for reasons I don't remember. =)